system_tests/language.py

```diff
@@ -88,7 +88,7 @@ def _check_analyze_entities_result(self, entities):
         self.assertEqual(entity2.metadata, {})
         # Verify entity 3.
         self.assertEqual(entity3.name, self.NAME3)
-        self.assertEqual(entity3.entity_type, EntityType.WORK_OF_ART)
+        self.assertEqual(entity3.entity_type, EntityType.EVENT)
```

Comment:
Like what JJ said last Monday, these ML tests change as the systems learn. So as long as they return something that is one of the expected responses, I think it's good.  So I'm 👍 for the `choices` approach.